src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -36,7 +36,7 @@
 #include <lib/support/CodeUtils.h>
 #include <lib/support/logging/CHIPLogging.h>
 
-#include <power/reboot.h>
+#include <sys/reboot.h>
```

Comment:
Can you post a PR upstream as well, or does upstream not integrate with this revision of Zephyr yet?

Reply:
Unfortunately we can't push this fixes to the upstream at the current state. We need to first finish Zephyr upmerge in NCS, and then bump NCS version in upstream Matter + include those fixes.

Reply:
The dependency between repositories is that NCS pulls proper version of Matter, but also Matter utilizes specific version of NCS, so we are not able to do this other way.